### PR TITLE
Replace cmp in acl_loader with operator.eq

### PR DESCRIPTION
**Pull request by ZhaohuiS**
Signed-off-by: Zhaohui Sun <zhaohuisun@microsoft.com>

#### What I did
When run try to add a new rule for existing rule table with this command `acl-loader update incremental`, it will throw a traceback like this:

```
admin@vlab-03:~$ sudo acl-loader update incremental /home/admin/external_acl.json 
Traceback (most recent call last):
  File "/usr/local/bin/acl-loader", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/acl_loader/main.py", line 1066, in incremental
    acl_loader.incremental_update()
  File "/usr/local/lib/python3.9/dist-packages/acl_loader/main.py", line 761, in incremental_update
    if cmp(self.rules_info[key], self.rules_db_info[key]) != 0:
NameError: name 'cmp' is not defined

```
cmp is deprecated in python3.

#### How I did it
1. Use `operator.eq` instead.
The return type of `operator.eq` is bool, it returns True if x is equal to y, False, otherwise.
`cmp` syntax:
`cmp(a, b)`
Parameters:
a and b are the two numbers in which the comparison is being done.
Returns:
-1 if a<b
0 if a=b
1 if a>b

2. Add unit test
#### How to verify it
Load acl json file with existing rule table in the config file with this command:
` acl-loader update incremental**.json`

### CODE REVIEW

**Review by bingwang-ms — DISMISSED** [review]

**Comment by bingwang-ms** [comment]
Please update test case to cover the diff.

**Review by prsunny — DISMISSED** [review]

File: acl_loader/main.py [hunk 1]
```diff
@@ -758,7 +759,7 @@ def incremental_update(self):
                 namespace_configdb.mod_entry(self.ACL_RULE, key, None)
 
         for key in existing_controlplane_rules:
-            if cmp(self.rules_info[key], self.rules_db_info[key]) != 0:
+            if operator.eq(self.rules_info[key], self.rules_db_info[key]) != 0:
```
Comment (qiluo-msft) [inline acl_loader/main.py:762]: `operator.eq` will return a bool?

Comment (ZhaohuiS) [reply acl_loader/main.py:762]: @qiluo-msft Yes. The return type of this method is bool, it returns True if x is equal to y, False, otherwise.

Comment (ZhaohuiS) [reply acl_loader/main.py:762]: @qiluo-msft Thank you for your question, for python2, cmp returns 0 if x is equal to y, if not equal, the result is not 0, it should be -1 or 1. So, operator.eq's return result is opposite with cmp's results. I will update code to make the logic right and write unit test case as well.

Syntax:
cmp(a, b)
Parameters:
a and b are the two numbers in which the comparison is being done. 
Returns:
-1 if a<b
0 if a=b
1 if a>b

**Review by bingwang-ms — DISMISSED** [review]

File: acl_loader/main.py (hunk 1 above)
Comment (qiluo-msft) [reply acl_loader/main.py]: I see your new iteration. To make it better, use `if not eq()`.

Comment (ZhaohuiS) [reply acl_loader/main.py]: @qiluo-msft Ye, agree, I updated it.

**Review by qiluo-msft — APPROVED** [review]

**Comment by prsunny** [comment]
@ZhaohuiS , we need this for 202012 as well, correct? Can you please raise a separate PR?

**Pull request merged**